test/falcor/get/get.dataSource-and-cache.spec.js

```diff
@@ -133,51 +133,37 @@ describe('DataSource and Partial Cache', function() {
         it('should ensure empty paths do not cause dataSource requests {from:1, to:0}', function(done) {
             var onGet = sinon.spy();
             var model = new Model({
-                cache: {
-                    a: Model.ref(['c']),
-                    c: {
-                        0: Model.atom('hello')
-                    }
-                },
-                source: {
-                    get: onGet
-                }
+                cache: { b: {} },
+                source: new LocalDataSource({}, { onGet: onGet })
             });
 
-            var modelGet = model.get(['b', {to:0, from:1}]);
+            var modelGet = model.get(['b', { from: 1, to: 0 }, 'leaf']);
```

Comment:
Updated these two tests that—with a little bit of tweaking—catch the original bug. The original intent of the tests remains intact. The tests now also give proper feedback on failure.